cluster/terraform_kubernetes/prometheus.tf

```diff
@@ -136,6 +135,7 @@ resource "kubernetes_deployment" "prometheus" {
             "sidecar",
             "--log.level=debug",
             "--tsdb.path=/prometheus",
+            "--web.enable-lifecycle",
```

Comment:
Can you move it back to line 86 to avoid the diff?

Reply:
Moved to rigt place now